dbcon/mysql/columnstore.cnf

```diff
@@ -1,7 +1,6 @@
 [mysqld]
 plugin-load-add=ha_columnstore.so
 plugin-maturity=beta
-lower_case_table_names=1
```

Comment:
I believe we need to leave setting here.

Reply:
We can't have this option enabled according to the server team.